test/ConfigProviderTest.php

```diff
@@ -10,23 +10,50 @@
 use PHPUnit\Framework\TestCase;
 use Zend\Di\ConfigProvider;
 use PHPUnit\Framework\Constraint\IsType;
+use Zend\Di\CodeGenerator\InjectorGenerator;
+use Zend\Di\InjectorInterface;
+use Zend\Di\ConfigInterface;
 
 /**
  * @coversDefaultClass Zend\Di\Module
  */
 class ConfigProviderTest extends TestCase
 {
-    public function testInstanceIsInvokable()
+    public function testInstanceIsInvokable(): void
     {
         $this->assertInternalType(IsType::TYPE_CALLABLE, new ConfigProvider());
     }
 
-    public function testProvidesDependencies()
+    public function testProvidesDependencies(): void
     {
         $provider = new ConfigProvider();
         $result = $provider();
 
         $this->assertArrayHasKey('dependencies', $result);
         $this->assertEquals($provider->getDependencyConfig(), $result['dependencies']);
     }
+
+    /**
+     * Provides service names that should be defined with a factory
+     */
+    public function provideExpectedServicesWithFactory(): iterable
+    {
+        return [
+            //               service name
+            'injector'  => [ InjectorInterface::class ],
+            'config'    => [ ConfigInterface::class ],
+            'generator' => [ InjectorGenerator::class ],
+        ];
+    }
+
+    /**
+     * Test if a service is provided by factory definition
```

Comment:
Not sure if we need description here. Is it giving us more information than just name of test method?